projects/distributed-tracing/src/shared/dashboard/data/graphql/waterfall/trace-waterfall-data-source.model.ts

```diff
@@ -62,9 +71,10 @@ export class TraceWaterfallDataSourceModel extends GraphQlDataSourceModel<Waterf
       requestType: TRACE_GQL_REQUEST,
       traceId: this.traceId,
       spanLimit: 1000,
-      timeRange: this.getTimeRangeOrThrow(),
+      timeRange: this.getTraceTimeRangeOrThrow(),
       traceProperties: [],
-      spanProperties: this.spanSpecifications
+      spanProperties: this.spanSpecifications,
+      spansTimeRange: this.getTimeRangeOrThrow()
```

Comment:
Previously span time range was same as Selected time range. Keeping it the same now. Although, it is definitely incorrect. We should be able to fetch all spans here for the given trace.

Reply:
Ah, commented on this above - we should be able to fix this.